src/DiseaseParm.cpp

```diff
@@ -65,6 +68,9 @@ void DiseaseParm::Initialize ()
 
     // Multiply contact rates by transmission probability given contact
     xmit_work *= p_trans[0];
+    xmit_work_diff_age *= p_trans[0];
+    xmit_work_diff_group *= p_trans[0];
+    xmit_work_diff_group_diff_age *= p_trans[0];
```

Comment:
Do these lines belong to this PR? Or is this from the work you are doing with improving workplace interactions?